.github/workflows/test.yml

```diff
@@ -4,30 +4,31 @@ jobs:
   test:
     strategy:
       matrix:
-        rust-toolchain: [1.49.0]
+        rust-toolchain: [stable]
         platform: [ubuntu-latest]
     runs-on: ${{ matrix.platform }}
+    environment: test
     steps:
     - name: Checkout code
-      uses: actions/checkout@v1
+      uses: actions/checkout@v3
     - name: Install Rust
       uses: actions-rs/toolchain@v1
```

Comment:
NOTE: This action is no longer maintained.

I've switched the CLI to `uses: dtolnay/rust-toolchain@stable`.